### PR TITLE
Set development version to 1.9.0dev

### PR DESCRIPTION
**Pull request by awaelchli**
## What does this PR do?

See title.

## Before submitting
- [x] Was this discussed/approved via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure to update the documentation with your changes? (if necessary)
- [x] Did you write any new necessary tests? (not for typos and docs)
- [x] Did you verify new and existing tests pass locally with your changes?
- [x] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [x] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?
I made sure I had fun coding 🙃

cc @borda

### CODE REVIEW

**Review by carmocca — APPROVED** [review]
Pushed a missing change

File: src/lightning/__version__.py [hunk 1]
```diff
@@ -1 +1 @@
-version = "1.8.0"
+version = "1.8.0dev"
```
Comment (Borda) [inline src/lightning/__version__.py:1]:
```suggestion
version = "1.9.0dev"
```

File: src/lightning_app/__version__.py [hunk 2]
```diff
@@ -1 +1 @@
-version = "1.8.0"
+version = "1.8.0dev"
```
Comment (Borda) [inline src/lightning_app/__version__.py:1]:
```suggestion
version = "1.9.0dev"
```

File: src/pytorch_lightning/__version__.py [hunk 3]
```diff
@@ -1 +1 @@
-version = "1.8.0"
+version = "1.8.0dev"
```
Comment (Borda) [inline src/pytorch_lightning/__version__.py:1]:
```suggestion
version = "1.9.0dev"
```

**Comment by awaelchli** [comment]
@Borda 🤣 🤣 🤣 🤣

**Review by rohitgr7 — APPROVED** [review]

**Review by Borda — APPROVED** [review]

**Review by manskx — APPROVED** [review]

**Pull request merged**